docs/spec_draft.md

```diff
@@ -127,3 +127,43 @@ specification.
 %z = stablehlo.add %x, %y : tensor<2x2xf32>
 // %z: [[6, 8], [10, 12]]
 ```
+
+## stablehlo.constant
+
+`stablehlo.constant(value) -> result`
+
+### Semantics
+
+Creates a constant value or array from `value` and produces a `result` tensor.
+
+### Attributes
+
+| Name | Type |
+|-|-|
+| `value` | `tensor` of Integer, Floating-point and Complex element types |
```

Comment:
How about saying "tensor of any supported element type" to be more concise? I know this is how `stablehlo.add` is currently specced, and we have a few more specs in the pipeline, so perhaps this is not the right venue for discussing this wording. /cc @sdasgup3